includes/Performance.php

```diff
@@ -360,4 +362,17 @@ public function isJetPackBoostActive() {
 
 		return $exists;
 	}
+
+	/**
+	 * Set default values for JetPack Boost.
+	 *
+	 * @return void
+	 */
+	public function set_default_values_for_jetpack_boost() {
+		$isFreshInstall = container()->get( 'isFreshInstallation' );
```

Comment:
```suggestion
	        if ( container()->has( 'isFreshInstallation' ) ) {
			$isFreshInstall = container()->get( 'isFreshInstallation' );
		}
```

We should probably do a check like this, as the default behavior is to throw an Exception when it does not exist causing a fatal error.

Reply:
ok